warp/Network/Wai/Handler/Warp/Windows.hs

```diff
@@ -18,7 +19,9 @@ windowsThreadBlockHack :: IO a -> IO a
 windowsThreadBlockHack act = do
     var <- newEmptyMVar :: IO (MVar (Either Control.Exception.SomeException a))
     -- Catch and rethrow even async exceptions, so don't bother with UnliftIO
-    void . forkIO $ Control.Exception.try act >>= putMVar var
+    void . forkIO $ do
```

Comment:
I think we can keep the `labelThread` outside of the forked thread.

```hs
...
threadId <- forkIO $ Control.Exception.try act >>= putMVar var
labelThread threadId "Windows Thread Block Hack (warp)"
...
      
```

Reply:
Done